apps/tup-cms/src/apps/portal/apps.py

```diff
@@ -44,18 +44,26 @@ def submit_ticket(form_data):
 
 
 def send_confirmation_email(form_name, form_data):
+
+    tour_receipt = ""
+    if form_name == "Tour Request Form":
```

Comment:
**Musing.** I thought, "Should we use a different, less volatile, identifier". But form ID is less intuitive, and name is unlikely to change. It might even be more likely that the form is accidentally deleted and recreated… Coupling this logic to form name means such a problem case could be fixed by admin.